src/config.coffee

```diff
@@ -32,6 +32,11 @@ module.exports = {
         type : 'boolean'
         default : true
         description : "When beautifying on save, use the entire file, even if there is selected text in the editor. Important: The `beautify on save` option for the specific language must be enabled for this to be applicable. This option is not `beautify on save`."
+      beautifyFileExtensions :
```

Comment:
Change `beautifyFileExtensions` to `excludeFileExtensions` or something more accurate.